ui/page/transaction/transaction_details_page.go

```diff
@@ -956,6 +947,18 @@ func (pg *TxDetailsPage) pageSections(gtx C, body layout.Widget) D {
 // displayed.
 // Part of the load.Page interface.
 func (pg *TxDetailsPage) HandleUserInteractions(gtx C) {
+
+	for _, item := range pg.moreItems {
```

Comment:
please check `pg.moreItems`,  I don' think it need a slice.

Reply:
please refactor it

Reply:
yes, I will check it

Reply:
If we don’t need more button, I will move “view on block explorer” button to transaction detail page and remove “…” button.

Reply:
@dreacot I think we should move `view on block explorer` button to outside like proposal page...

Reply:
Ok, I will do it

Reply:
@dreacot Check out this conversation

Reply:
i see this now, my question remains though, what happens to that view on mobile

Reply:
from the comments above , i see.a series of misunderstanding, @JustinBeBoy did not say the more button wasn't needed, he said he doesn't think it should be a slice

Reply:
please revert

Reply:
ok, the "View on block explorer" button has been returned to the "more options" button @dreacot